packages/elements/src/components/ino-chip/ino-chip.scss

```diff
@@ -15,6 +15,18 @@ ino-chip {
   .mdc-chip {
     @include chips.fill-color-accessible($default-chip-color);
 
+    .mdc-chip__icon--trailing {
+      width: 20px;
+      height: 20px;
+      font-size: 20px;
+    }
+
+    ino-icon {
```

Comment:
This seems to be a duplication of lines 24 - 28, no?

Reply:
apparently 😄